src/main/java/org/myrobotlab/service/InMoov2.java

```diff
@@ -2132,48 +2244,7 @@ public void startedGesture(String nameOfGesture) {
   }
 
   public void startHeartbeat() {
-    addTask(1000, "publishHeartbeat");
```

Comment:
this time should be configurable or pulled out as a constant for the interval here..